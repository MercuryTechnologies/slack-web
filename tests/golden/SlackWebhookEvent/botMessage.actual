EventEventCallback
    ( EventCallback
        { eventId = EventId
            { unEventId = "Ev043VFLT545" }
        , teamId = TeamId
            { unTeamId = "T043DB835ML" }
        , eventTime = MkSystemTime
            { systemSeconds = 1664216870
            , systemNanoseconds = 0
            }
        , event = EventMessage
            ( MessageEvent
                { blocks =
                    [ RichText
                        { blockId = Just
                            ( NonEmptyText "FVI" )
                        , elements =
                            [ RichTextSectionItemRichText
                                [ RichItemText "TEST"
                                    ( RichStyle
                                        { rsBold = False
                                        , rsItalic = False
                                        }
                                    )
                                ]
                            ]
                        }
                    ]
                , channel = ConversationId
                    { unConversationId = "C043YJGBY49" }
                , text = "TEST"
                , channelType = Channel
                , user = UserId
                    { unUserId = "U0442US8QGH" }
                , ts = "1664216870.477049"
                , threadTs = Just "1664216109.798919"
                , appId = Just "A0442TUPHGR"
                , botId = Just "B0439P161B9"
                }
            )
        }
    )