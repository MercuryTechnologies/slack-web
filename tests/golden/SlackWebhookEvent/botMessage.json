{
  "token": "aaaa",
  "team_id": "T043DB835ML",
  "api_app_id": "A0442TUPHGR",
  "event": {
    "bot_id": "B0439P161B9",
    "type": "message",
    "text": "TEST",
    "user": "U0442US8QGH",
    "ts": "1664216870.477049",
    "app_id": "A0442TUPHGR",
    "team": "T043DB835ML",
    "bot_profile": {
      "id": "B0439P161B9",
      "deleted": false,
      "name": "Slacklinker dev",
      "updated": 1663960005,
      "app_id": "A0442TUPHGR",
      "icons": {
        "image_36": "https://a.slack-edge.com/80588/img/plugins/app/bot_36.png",
        "image_48": "https://a.slack-edge.com/80588/img/plugins/app/bot_48.png",
        "image_72": "https://a.slack-edge.com/80588/img/plugins/app/service_72.png"
      },
      "team_id": "T043DB835ML"
    },
    "blocks": [
      {
        "type": "rich_text",
        "block_id": "FVI",
        "elements": [
          {
            "type": "rich_text_section",
            "elements": [
              {
                "type": "text",
                "text": "TEST"
              }
            ]
          }
        ]
      }
    ],
    "thread_ts": "1664216109.798919",
    "parent_user_id": "U043H11ES4V",
    "channel": "C043YJGBY49",
    "event_ts": "1664216870.477049",
    "channel_type": "channel"
  },
  "type": "event_callback",
  "event_id": "Ev043VFLT545",
  "event_time": 1664216870,
  "authorizations": [
    {
      "enterprise_id": null,
      "team_id": "T043DB835ML",
      "user_id": "U0442US8QGH",
      "is_bot": true,
      "is_enterprise_install": false
    }
  ],
  "is_ext_shared_channel": false,
  "event_context": "aaaa"
}
